app/views/guidance/landing-page.njk

```diff
@@ -0,0 +1,109 @@
+{% extends './layouts/layout.njk' %}
+
+{% block pageTitle %}Guidance for farmers - {{ serviceName }}{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <div>
+        <span class="govuk-caption-xl">Collection</span>
+        <h1 class="govuk-heading-xl">Annual health and welfare review of livestock</h1>
+      </div>
+      <p class="govuk-body govuk-!-font-size-24">
+      Farmers who keep cattle, sheep and pigs can get funding to pay for a vet to visit their farm and carry out an annual health and welfare review.
+      </p>
+      <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+      <p class="govuk-body-s">From: <a class="govuk-link govuk-!-font-weight-bold" href="https://www.gov.uk/government/organisations/department-for-environment-food-rural-affairs">Department for Environment, Food &amp; Rural Affairs</a> and <a class="govuk-link govuk-!-font-weight-bold" href="https://www.gov.uk/government/organisations/rural-payments-agency">Rural Payments Agency</a></p>
+    </div>
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body">
+      A health and welfare review of livestock provides funding for a vet, or team chosen by a vet, to visit a farm and carry out a health and welfare review of beef or dairy cattle, sheep or pigs.
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>reduce endemic diseases and conditions</li>
+      <li>increase animal productivity</li>
+      <li>improve animal welfare</li>
+    </ul>
+    <p class="govuk-body">
+      The review is optional. It is not an inspection and will not be used for compliance or monitoring.
+    </p>
+    <h2 class="govuk-heading-m">Farmers</h2>
+    <p class="govuk-body">Read the apply guidance to find out:</p>
```

Comment:
According to the word document, this should read 

`Read the apply guidance to check:`